app/Services/DirectAdmin/EmailListAdapter.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services\DirectAdmin;
+
+use App\Services\DirectAdmin\Contracts\Client;
+use App\Services\DirectAdmin\ValueObjects\Command;
+use App\Services\DirectAdmin\ValueObjects\EmailList;
+use App\Services\DirectAdmin\ValueObjects\EmailListSubscribers;
+
+final class EmailListAdapter implements Contracts\EmailListAdapter
```

Comment:
Dit voelt als een DA specifieke implementatie, dus zou je dit niet beter `DAEmailListAdapter` oid noemen? Of vind je de namespace voldoende